bin/check_tron_jobs.py

```diff
@@ -112,12 +115,24 @@ def pretty_print_actions(action_run):
     return display.format_action_run_details(action_run)
 
 
-def get_relevant_run(job_runs):
+def get_relevant_run_and_state(job_runs):
+    if len(job_runs['runs']) == 0:
+        return None, "no_run_yet"
+    run = is_job_scheduled(job_runs)
+    if run is None:
+        return job_runs['run'][0], "not_scheduled"
     run = is_job_stuck(job_runs)
     if run is not None:
-        return run
+        return run, "stuck"
```

Comment:
It looks like it will be very easy to break the "setting" of these magic strings and the interpretation of these strings in the other function. Silly typos can easily break this code.

Can you use an ENUM to eliminate this type of bug?
https://docs.python.org/3/library/enum.html